src/OpenyActivityFinderBackend.php

```diff
@@ -161,6 +185,62 @@ public function getPartsOfDay() {
     ];
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getStartMonths() {
+    return [
+      [
+        'label' => $this->t('January'),
+        'value' => '1',
+      ],
+      [
+        'label' => $this->t('February'),
+        'value' => '2',
+      ],
+      [
+        'label' => $this->t('March'),
+        'value' => '3',
+      ],
+      [
+        'label' => $this->t('April'),
+        'value' => '4',
+      ],
+      [
+        'label' => $this->t('May'),
+        'value' => '5',
+      ],
+      [
+        'label' => $this->t('June'),
+        'value' => '6',
+      ],
+      [
+        'label' => $this->t('July'),
+        'value' => '7',
+      ],
+      [
+        'label' => $this->t('August'),
+        'value' => '8',
+      ],
+      [
+        'label' => $this->t('September'),
+        'value' => '9',
+      ],
+      [
+        'label' => $this->t('October'),
+        'value' => '10',
+      ],
+      [
+        'label' => $this->t('November'),
+        'value' => '11',
+      ],
+      [
+        'label' => $this->t('December'),
+        'value' => '12',
+      ],
+    ];
+  }
+
```

Comment:
Could be something like this:
```
/**
 * {@inheritdoc}
 */
public function getStartMonths() {
  $months = [];

  for ($month = 1; $month <= 12; $month++) {
    $months[] = [
      'label' => $this->t(date('F', mktime(0, 0, 0, $month, 1))),
      'value' => (string) $month,
    ];
  }

  return $months;
}
```

Reply:
@rollki yes, it could be this way, but for what? It's simple data. Why should we calculate it every time? Sometimes, a simple solution is better. But I can be wrong, of course :)

Reply:
In general, I think it can be done even easier, probably the best solution will be to use the existing functionality of the core.
The core already has a `DateHelper` that has a `monthNames` method.
It will return an array of `TranslatableMarkup` objects, which is probably fine for us.

`$monthOptions = DateHelper::monthNames(TRUE);`

![Screenshot 2024-04-23 at 11 25 40](https://github.com/YCloudYUSA/yusaopeny_activity_finder/assets/50984627/6bd5d840-051d-4a12-9d8d-30093c5ad4a9)

cc @podarok @aleevas